pairol_alex/Assignments/PartII.py

```diff
@@ -0,0 +1,7 @@
+for x in range(0, 1000):
+    if (x % 2 !=0 ):
+        print x
+
+
+for x in range(0, 1000,5):
```

Comment:
Good job. It's easy for instructors to identify those that read the documentation =]